src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -627,6 +643,10 @@ export class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang) {
   // ########### VUEX ###############
   @State(({trees: {tree}}) => tree) tree!: any
   @State(({trees: {ui}}) => ui) ui!: any
+  @State((state) => state.trees.ui.toolbar.hasFlowTitle) hasFlowTitle!: boolean
```

Comment:
Similar to my comment on ImportFlow. How about a getter which knows about the structure of the config file?

```
getters: {  
  getToolbarConfig: ({ui}, key) => lodash.get(ui.toolbar, key, null)
}
```